### DISCUSSION TITLE
Branching Strategy ALZ

### DISCUSSION THREAD

**Discussion by FallenHoot** [Azure/Enterprise-Scale#594] [General]
A “branching strategy” refers to the strategy a software development team employs when writing, merging, and shipping code in the context of a version control system like Git.

Currently, ALZ doesn't address this as it is very opinionated.

Should guidance around using one of the methods below be documented within the wiki?:
[GitFlow](https://nvie.com/posts/a-successful-git-branching-model/)
[GitHub Flow](https://guides.github.com/introduction/flow/)
[Trunk-based development](https://trunkbaseddevelopment.com/)

**Comment by Springstone** [comment]
Closing this discussion, as we have a clear branching strategy for programs, and require forks to merge with main.
